core/src/main/resources/error/error-classes.json

```diff
@@ -1967,6 +1984,11 @@
       "The seed expression <seedExpr> of the expression <exprWithSeed> must be foldable."
     ]
   },
+  "SET_OPERATION_ON_MAP_TYPE_UNSUPPORTED" : {
```

Comment:
Technically, it can be supported, right? Could you add a sub-class to `UNSUPPORTED_FEATURE`.